src/index.js

```diff
@@ -756,6 +783,19 @@ class CgSlider extends EventEmitter {
     maxHandle.setAttribute('aria-valuemax', this.max);
   }
 
+  /**
+   * Update aria-valuetext attributes for both handles
+   * @private
+   * @param {number} valMin Min handle value
+   * @param {number} valMax Max handle value
+   */
+  _updateValueTexts(valMin, valMax) {
```

Comment:
Current method should be renamed to `_updateAriaValueTexts` to be consistent with other methods related to aria-attributes.